content/en/docs/concepts/policy/limit-range.md

```diff
@@ -31,17 +31,78 @@ LimitRange object in that namespace.
 The name of a LimitRange object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
-### Overview of Limit Range
+### Overview of LimitRange
 
 - The administrator creates one LimitRange in one namespace.
 - Users create resources like Pods, Containers, and PersistentVolumeClaims in the namespace.
-- The `LimitRanger` admission controller enforces defaults and limits for all Pods and Containers that do not set compute resource requirements and tracks usage to ensure it does not exceed resource minimum, maximum and ratio defined in any LimitRange present in the namespace.
-- If creating or updating a resource (Pod, Container, PersistentVolumeClaim) that violates a LimitRange constraint, the request to the API server will fail with an HTTP status code `403 FORBIDDEN` and a message explaining the constraint that have been violated.
-- If a LimitRange is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
+- First, the `LimitRange` admission controller applies default request and limit values for all Pods and Containers that do not set compute resource requirements.
+- Second, the `LimitRange` tracks usage to ensure it does not exceed resource minimum, maximum and ratio defined in any `LimitRange` present in the namespace.
+- If creating or updating a resource (Pod, Container, PersistentVolumeClaim) that violates a `LimitRange` constraint, the request to the API server will fail with an HTTP status code `403 FORBIDDEN` and a message explaining the constraint that have been violated.
+- If a `LimitRange` is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
   requests or limits for those values. Otherwise, the system may reject Pod creation.
-- LimitRange validations occurs only at Pod Admission stage, not on Running Pods.
-
-Examples of policies that could be created using limit range are:
+- `LimitRange` validations occurs only at Pod Admission stage, not on Running Pods.
+
+{{< note >}}
```

Comment:
This note is too big. Consider making it a subtopic with a heading instead. Also, please use a `{{< codenew >}}` shortcode for the LimitRange manifest.

Reply:
fixed